cli/pcluster/update/update_command.py

```diff
@@ -206,3 +207,18 @@ def _restore_desired_size(cfn_params, stack_name, scheduler):
     # If there are compute nodes running, preserve current desired capacity (if possible)
     if int(cfn_params["MinSize"]) <= desired_capacity <= int(cfn_params["MaxSize"]):
         cfn_params["DesiredSize"] = str(desired_capacity)
+
+
+def _check_cluster_models(base_config, target_config):
+    if target_config.cluster_model == ClusterModel.SIT:
+        if base_config.cluster_model == ClusterModel.HIT:
+            LOGGER.error(
+                "The configuration file '{config_file}' must be converted to the latest format with support for "
```

Comment:
The message should be related to the section not to the file.
Because in the same file you can have other sections old-style that you're not using.

Reply:
That's an interesting point. @nstornetta how would you modify the message in this way?

Reply:
I modified the message to: "The configuration of the cluster section in the '{config_file}'"